tests/light/src/axosyslog_light/helpers/loggen/loggen_executor.py

```diff
@@ -91,6 +92,8 @@ def format(self) -> typing.List[str]:
             params.append("--loop-reading")
         if self.rate is not None:
             params.append(f"--rate={self.rate}")
+        if self.rate_burst_start:
```

Comment:
Could you please add `--perf` as a new supported option?

Reply:
I've added it.